reference/tax_settings.v3.yml

```diff
@@ -141,6 +141,9 @@ components:
             - FIXED
             - BASIC
             - DISABLE
+        tax_should_unwind_store_tax:
+          type: boolean
+          description: Determines tax unwinding behaviour where when active, it takes away the store tax rate and applies the shopper tax rate.
```

Comment:
The experiment description is as follows:
`When enabled, we avoid rounding during tax unwinding processes`
I would use something similar to this.

Reply:
updated 👍